src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Let's make code more readable and split this method into smaller ones. For example you can create methods
    private String createResultString(int buy, int supply) and private void writeToFile(String fileName, String text). They will be called in gtStatistic